.github/workflows/issue-release-workflow.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           printf "%s" "$COMMENT_BODY" |
           ./llvm/utils/git/github-automation.py \
           --repo "$GITHUB_REPOSITORY" \
-          --token ${{ github.token }} \
+          --token "${{ secrets.RELEASE_WORKFLOW_PR_CREATE }}" \
```

Comment:
Just to double check, does this token also have permission to create issue comments? It will be used for that as well.

Reply:
I just added issue permissions to the token.